SolrNet.Tests/ExceptionTests.cs

```diff
@@ -25,8 +27,11 @@ public class ExceptionTests {
         public void All_exceptions_are_serializable() {
             var allExceptions = typeof (SolrNetException).Assembly.GetTypes().Where(t => typeof (SolrNetException).IsAssignableFrom(t));
             foreach (var e in allExceptions) {
-                Assert.IsAssignableFrom<System.Runtime.Serialization.ISerializable>(e);
+                if (!typeof(ISerializable).IsAssignableFrom(e))
+                {
+                    throw new Exception($"{e} is not serializable"); 
+                }
```

Comment:
I guess the `Assert.IsAssignableFrom` didn't do the correct job. However perhaps instead of throwing an exception, you can just Assert
```suggestion
                Assert.True (typeof(ISerializable).IsAssignableFrom(e));           
```

Reply:
Yep `Assert.IsAssignableFrom` expects an _instance_ of the exception but we were passing a type here, so it wasn't what we wanted :slightly_smiling_face: 

About `Assert.True (typeof(ISerializable).IsAssignableFrom(e));`, that works too, but does that print out the exception type in case of a assertion failure?

Reply:
True that, this should be the combination of both worlds: `Assert.True (typeof(ISerializable).IsAssignableFrom(e),  $"{e} is not serializable");`

Reply:
:+1: Applied in https://github.com/SolrNet/SolrNet/pull/535/commits/d4ce6798afcc03cd5f8dc921e85e70e65d08620a#diff-40a4ac6cbfa81603507e85625ecb2ac76c9381f6a53ee25f454503975c8ee73cR30